lib/resources/merchants.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var merge = require('../merge');
+var formatQuery = require('../formatQuery');
+
+function merchants(requestOptions, maybePromiseRequest) {
+  return {
+    all: function all(params, callback) {
+      //
+      // Gets an index of all merchants.
```

Comment:
I'd suggest listing the options under @params too, so someone reading just this file would know that filtering options are available.